.github/workflows/release.yml

```diff
@@ -0,0 +1,136 @@
+name: Release and Propose an Upgrade
+
+env:
+  SUBWASM_VERSION: 0.20.0
+
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - 'main'
+
+jobs:
+  srtool:
+    if: ${{ github.event.pull_request.merged == true && contains(github.event.pull_request.title, 'release:') }}
+    name: SrTool check on ${{ matrix.chain }} ${{ github.event.inputs.ref }}
+    strategy:
+      fail-fast: false
+      matrix:
+        chain: ["amplitude", "pendulum", "foucoco"]
+        shouldReleaseAmp:
+          - ${{ contains(github.event.pull_request.title, 'amplitude') }}
+        shouldReleasePen:
+          - ${{ contains(github.event.pull_request.title, 'pendulum') }}
+        shouldReleaseFou:
+          - ${{ contains(github.event.pull_request.title, 'foucoco') }}
+        exclude:
+          - shouldReleaseAmp: false
+            chain: "amplitude"
+          - shouldReleasePen: false
+            chain: "pendulum"
+          - shouldReleaseFou: false
+            chain: "foucoco"
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.inputs.ref }}
+          fetch-depth: 0
+
+      - name: Srtool build
+        id: srtool_build
+        uses: chevdor/srtool-actions@v0.9.2
+        with:
+          chain: ${{ matrix.chain }}
+          runtime_dir: runtime/${{ matrix.chain }}
+          # this is important to avoid build fail. See https://github.com/paritytech/srtool/issues/62
+          tag: 1.66.1
+      - name: Summary
+        run: |
+          echo '${{ steps.srtool_build.outputs.json }}' | jq > ${{ matrix.chain }}-srtool-digest.json
+          cat ${{ matrix.chain }}-srtool-digest.json
+          echo "Runtime location: ${{ steps.srtool_build.outputs.wasm }}"
+          echo '${{ steps.srtool_build.outputs.json }}'
+          cp ${{ steps.srtool_build.outputs.wasm }} ${{ matrix.chain }}_runtime.compact.wasm
```

Comment:
this will copy the file found deeply nested in directories, to the current directory.